lib/mayaUsd/fileio/utils/meshReadUtils.cpp

```diff
@@ -919,8 +974,35 @@ UsdMayaMeshReadUtils::getComponentTags(const UsdGeomMesh& mesh, std::vector<Comp
         compFn.addElements(mFaces);
 
         tags.emplace_back(tagName, faceComp);
+
+        // Gather roundtrip data:
+        JsObject subsetRoundtripData;
+
+        TfToken familyName;
+        ss.GetFamilyNameAttr().Get(&familyName);
+        if (familyName != UsdMayaGeomSubsetTokens->ComponentTagFamilyName) {
+            subsetRoundtripData[ss.GetFamilyNameAttr().GetBaseName()]
+                = JsValue(familyName.GetString());
+        }
+
+        // Could be any applicable API, but we mostly care about materials:
```

Comment:
Not sure what that comment implies: is it that we could extend the work to other things?

Reply:
Changed to a less ambiguous comment, but you can imagine some very technical users adding new Schema APIs, custom data, attributes, to GeomSubsets. Not sure we want to blindly roundtrip everything, so let's concentrate on materials since they have a representation in Maya.